supabase/migrations/20240503131452_rename_request_denied_to_parcel_denied.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE status_order
```

Comment:
A duplicate caused because: I wrote it incorrectly the first time and it didn't do anything (I got 'parcel denied' and 'request denied' the wrong way round), then I edited this file but it didn't seem to update - I guess because it wasn't a new migration? - so I added another one. 
Be grateful to hear if there is a better way of dealing with this :)

Reply:
@kaydentsoftwire you can just delete the migration file and recreate a new one

Reply:
Thank you :)